docs/section5/Rule5-7-1.md

```diff
@@ -0,0 +1,36 @@
+# Envelope - Rule 5-7-1 
+**Rule ID:** 5-7-1  
+**Rule Description:** Baseline WWR should be equal to User WWR or 40%, whichever is smaller.  
+**Rule Assertion:** Baseline RMR = expected value  
+**Appendix G Section:** Envelope  
+**Appendix G Section Reference:**
+- Table G3.1, 5. Building Envelope, Baseline Building Performance, c. Vertical Fenestration Areas
+- Table G3.1.1-1
+
+**Data Lookup:** Table G3.1.1-1  
+**Evaluation Context:**  Each Data Element  
+
+**Applicability Checks:** 
+1. Building has spaces that are NEW, ADDITION or ALTERATION.
+
+**Manual Checks:** None  
+
+## Rule Logic:
+- **Applicability Check 1:** `length( [ if _space.status_type for _space in U_RMR...spaces is in [NEW, ADDITION, ALTERATION] ] ) > 0:`  
+- For each building segment in the Baseline model: `For _building_segment in B_RMR.building.building_segments:`
+    - Get the building area type of the building segment: `_building_area_type = _building_segment.area_type_vertical_fenestration`
+    - If building area type exists in Table G3.1.1-1 then get the allowable WWR for the building segment: `if _building_area_type in table_G3_1_1_1: _allowable_baseline_wwr = data_lookup(table_G3_1_1_1, _building_area_type) else: _allowable_baseline_wwr = 0.40`
```

Comment:
Either way, we get the allowable WWR, so I think line 22 should start with:   
"Get the allowable WWR for the building segment: "